.github/pull_request_template.md

```diff
@@ -10,7 +10,7 @@ _Add more context to describe the changes..._
 
 [Example](www.google.com)
 
-### Checks
+### Required Checks
```

Comment:
Good idea

Reply:
Or people don't do all the checks that are required